person.rb

```diff
@@ -20,4 +20,8 @@ def of_age?()
   def can_use_service?
     of_age? || @parents_permissiom
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] This Person class should inherit from the Nameable class. Kindly import the Nameable class and fix this.

  ```
  require_relative './nameable'

  class Person < Nameable
    ...
  ```